platform/openide.loaders/src/org/openide/awt/Toolbar.java

```diff
@@ -436,7 +437,12 @@ protected Object createInstance(final InstanceCookie[] cookies)
                     }
                     if (obj instanceof Action) {
                         Action a = (Action) obj;
-                        JButton b = new DefaultIconButton();
+                        AbstractButton b;
+                        if (a.getValue(Actions.ACTION_VALUE_TOGGLE) != null) {
```

Comment:
@sdedic, this is related to your toggle button changes. Can you review this part?